.github/workflows/benchmarks.yml

```diff
@@ -50,133 +58,53 @@ jobs:
           cmake .. -DCMAKE_BUILD_TYPE='Release' -DSOLC_VERSION_ZKEVM='1.0.0-lts' -DPEDANTIC=OFF -DCMAKE_CXX_FLAGS='-w' -DUSE_Z3=OFF -DUSE_CVC4=OFF
           make -j$(nproc)
 
+      - name: Build LLVM
+        uses: ./.github/actions/build-llvm
+
       - name: Benchmarking the Solidity compiler candidate
```

Comment:
Is it candidate or any?

Reply:
It is any, the LLVM build is always. the same here. I thought the thing in this repository is to test the performance impact of `solc` executable, everything else should stay unchanged. At least, it is how it was before, no changes here from my side.

@hedgar2017  could you, please, confirm that it's the right behavior?

Reply:
@antonbaliasnikov yes, only the `solc` executable should be changed between `reference` and `candidate` for PRs, but for manual workflows you should also be able to specify the compiler-tester branch and other options that may cause friction if hardcoded.

My comment was regarding the word `candidate` found several times in this job. To me it looks like the job is generic over reference and candidate, but below I can only see `--benchmark='candidate.json'` and no job to get the `reference.json`. Could you clarify how the job is supposed to work?

Reply:
@hedgar2017 ,
Got you. This is one line I missed to make generic. Fixed now and added a capability to choose the `compiler-tester` branch as we have in other workflows.

Could you, please, check again? Should be good now.